tools/generate_json_rpc_docs.py

```diff
@@ -136,7 +133,7 @@ def to_markdown(self):
         output = []
         output.append("| Name | Type | Description |")
         output.append("| --- | --- | --- |")
-        tag_re = re.compile(r"^\{(\w+)\}\s+([\S]+)\s+-\s+(.*)$", re.DOTALL)
+        tag_re = re.compile(r"^\{(\w+)}\s+(\S+)\s+-\s+(.*)$", re.DOTALL)
```

Comment:
Why do we escape `{` but not `}` now? 

Note: Checking this on regexer or some comparable online site might be good finding any inconsistencies of the regexes.

Reply:
That's what the IDE linter picked up, but yes I think you're right - we can ditch both. I've tested and it's happy

```bash
# Original
$ cat /usr/include/qt/QtCore/qlocale.h | python -c 'import re; import sys; s = sys.stdin.read(); print(re.search(r"enum Country[^\n]+\{([^}]+)\}", s).group(1))' | md5sum
46625fcf574732033f70b2f10ce25bc4  -

# Without backslashes
cat /usr/include/qt/QtCore/qlocale.h | python -c 'import re; import sys; s = sys.stdin.read(); print(re.search(r"enum Country[^\n]+{([^}]+)}", s).group(1))' | md5sum
46625fcf574732033f70b2f10ce25bc4  -

```

Reply:
(Updated to remove both)

Reply:
Thanks.